src/Ionide.ProjInfo/Library.fs

```diff
@@ -91,29 +200,29 @@ module ProjectLoader =
                    "TargetFramework", tfm.Value
                if path.EndsWith ".csproj" then
                    "NonExistentFile", Path.Combine("__NonExistentSubDir__", "__NonExistentFile__")
-               "DotnetProjInfo", "true" 
+               "DotnetProjInfo", "true"
+               "MSBuildEnableWorkloadResolver", "false" // we don't care about workloads
                yield! globalProperties ]
 
 
     let buildArgs =
-        [| "ResolvePackageDependenciesDesignTime"
+        [| "ResolveAssemblyReferencesDesignTime"
```

Comment:
these are the full set of design time targets (taken from the msbuild docs as well as referencing buildalyzer)